### ISSUE TITLE
Deploy a new IIWBook-like site for the DIACC Identity Kit 

### ISSUE THREAD

**Issue by swcurran** [bcgov/identity-kit-poc#36]
Please deploy a new instance of a site that is similar to IIWBook but is for the DIACC Identity Kit. Users will go to this site after being issued their verifiedPerson credential.

Differences from IIWBook:

1. New Title/Wording on the front.
2. Display the QR code on the first page for the proof.
4. Use the verifiedPerson schema in the restrictions for the credential.
    - Do not ask for the email address, locality or street claims.
5. New wording on the "Here's what's happening..." screen.
6. After the verification of the proof, stop the processing - no IIWBook backend and issuance of a new credential.
7. Display a message "Hello <name>" and then list the claims collected in the proof.

That's the end of the demo.  No need to create all the extra items.

If we are able to do this in parallel with the work ongoing with the Admin and User apps for the IKP, it's OK to just use the verified email credential for the proof, and we'll update the code/config to use the other credential later.

Nice To Haves, but we need this fast:
- use the 1.0 Present Proof protocol instead of the v0.1 proof
- implement this as a new code base that is intended to be a starter kit for more deployments.
- use the aries-cloudcontroller-node instead of IIWBook
   - using this vs. IIWBook probably depends on who builds this.

**Comment by swcurran** [comment]
This is currently assigned to @seanadipose but depending on what @nrempel is up to we may ask him to do this work.

@esune - heads up that we will need to deploy this.

@watkinspd - FYI.

**Comment by swcurran** [comment]
Closing as a duplicate of #71.
